NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -552,17 +567,20 @@ def _filter_event(self, evtinfo, event_idx=None):
         -------
 
         skip: bool
-            Returns True to skip/reject event, return False to keep/read event
+            Returns False to skip/reject event, return True to keep/read event
         """
+        self._event_idx += 1
```

Comment:
are we not double counting here? I would prefer not having this counter in this function